### PR TITLE
feat(dsl): added extensions field to PostgresInfoForEnv to allow setting pg extensions

### PR DESCRIPTION
**Pull request by busla**
# create postgres extensions with DSL

https://github.com/island-is/island.is/issues/10903

## What
Grant developers to create postgres extensions for their databases using the DSL.

This is partially done (feature deployments). We need to upgrade to postgres 13.x for feature completion, see attached issue.  

## Why
Extensions need to be enabled per database but could be part of the self-service platform where db users with create perms could enable _trusted extensions_.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by brynjarorng — APPROVED** [review]
LGTM

File: apps/services/sessions/infra/sessions.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ const workerPostgresInfo = {
   username: 'services_sessions',
   name: dbName,
   passwordSecret: '/k8s/services-sessions/DB_PASSWORD',
+  extensions: ['uuid-ossp'],
```
Comment (saevarma) [inline apps/services/sessions/infra/sessions.ts:19]: Nice!
We should also add this to `apps/services/auth/ids-api/infra`

Comment (busla) [reply apps/services/sessions/infra/sessions.ts:19]: > Nice! We should also add this to `apps/services/auth/ids-api/infra`

done

**Comment by busla** [comment]
> Deployed services: .
> Excluded services: ``

why isn't session and ids-api affect?

**Pull request closed without merging**